test/server/cards/01_SOR/units/InfernoFourUnforgetting.spec.ts

```diff
@@ -0,0 +1,57 @@
+describe('Inferno Four - Unforgetting', function() {
+    integration(function(contextRef) {
+        describe('Inferno Four - Unforgetting\'s ability', function() {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['inferno-four#unforgetting'],
+                        deck: ['foundling', 'pyke-sentinel', 'atst', 'cartel-spacer', 'wampa'],
+                    },
+                    player2: {
+                        spaceArena: ['tie-advanced'],
+                    }
+                });
+            });
+
+            it('While playing/defeating lets you look at the top 2 cards of the deck and decide whether to put either them on the bottom or top of deck in any order.', function () {
```

Comment:
nit: test names shouldn't start with a capital letter since it will end up in the middle of a sentence: `Inferno Four - Unforgetting's ability While playing/defeating...`

Reply:
changed to lowercase

Reply:
Still uppercase looks like?

Reply:
you are right seems like i missed it then...my mistake